standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -1012,8 +1012,9 @@ struct CommitTxnRequest {
     3: optional list<WriteEventInfo> writeEventInfos,
     // Information to update the last repl id of table/partition along with commit txn (replication from 2.6 to 3.0)
     4: optional ReplLastIdInfo replLastIdInfo,
+    5: optional bool exclWriteEnabled = true,
```

Comment:
This should be added as the last parameter to not break backward compatibility no?

Reply:
we need to keep it consistent with downstream, see `HIVE-23759: refactor field order of CommitTxnRequest`